src/Config.js

```diff
@@ -25,7 +25,9 @@ var Config = function (root, configpath) {
     this.pluginsManager = new PluginsManager(this);  // Do we need back ref?
     this.emit('loaded');
     this.on('server:init', this.attachRoutes.bind(this));
-    this.parsed_opts = {};  // Default. TODO better option management.
+    this.parsed_opts = {
+        renderer: 'carto'
```

Comment:
Right, parseOptions is not called in the tests. I'll have a look at that on a second step.